crates/y-sweet-worker/src/config.rs

```diff
@@ -10,6 +10,7 @@ const CHECKPOINT_FREQ_SECONDS: &str = "CHECKPOINT_FREQ_SECONDS";
 const S3_ACCESS_KEY_ID: &str = "AWS_ACCESS_KEY_ID";
 const S3_SECRET_ACCESS_KEY: &str = "AWS_SECRET_ACCESS_KEY";
 const S3_REGION: &str = "AWS_REGION";
+const S3_ENDPOINT: &str = "S3_ENDPOINT";
 const S3_BUCKET_PREFIX: &str = "S3_BUCKET_PREFIX";
 const S3_BUCKET_NAME: &str = "S3_BUCKET_NAME";
```

Comment:
Despite the changes to the above env vars, I think we should leave these as `S3_` because they do not have AWS equivalents.